Marlin/Marlin_main.cpp

```diff
@@ -117,7 +117,8 @@
  * M100 - Watch Free Memory (for debugging) (Requires M100_FREE_MEMORY_WATCHER)
  * M104 - Set extruder target temp.
  * M105 - Report current temperatures.
- * M106 - Fan on.
+ * M106 - Fan on: S0-255 : T3-255 / T2 / T1 Temporary set speed during printing
```

Comment:
Please move documentation like this to the `M106` handler. These descriptions are intentionally terse.